tsconfig.base.json

```diff
@@ -32,22 +33,5 @@
     "skipLibCheck": true,
     "stripInternal": true,
     "resolveJsonModule": true
-  },
-  "typedocOptions": {
```

Comment:
currious why this isn't neccesary?

Reply:
It's not _necessary_ because we're not using TypeDoc anymore. But I'm not fully ripping that out right now, and I don't remember now why I felt it was helpful to get rid of this. Something about the way the TS configs inherit from each other? I probably could add it back, but we're just going to rip it out again shortly when we completely remove TypeDoc and Docusaurus as unused.